test/functional/content_view/version/incremental_update_test.rb

```diff
@@ -88,4 +88,18 @@
     result = run_cmd(@cmd + params)
     assert_equal(result.exit_code, 0)
   end
+
+  it "performs incremental update with packages" do
+    params = ['--content-view-version-id=5', '--package-ids=1', '--lifecycle-environment-ids=1']
```

Comment:
We want to test the `--packages` option specifically rather than `--package-ids`. This will create another request to `packages#index`.